services/init-protonet.service

```diff
@@ -33,6 +34,7 @@ ExecStartPre=/bin/chown 1000:1000 /data/soul/log
 ExecStartPre=/bin/chown 1000:1000 /data/soul/public
 ExecStartPre=/bin/chown 1000:1000 /data/soul/uploads
 ExecStartPre=/bin/chown 1000:1000 /data/soul/protosync
+ExecStartPre=/bin/chown 1000:1000 /data/fonts
```

Comment:
The docker-image on the other hand is called `platform-unoconv`, is public, and at least in theory could be used by other clients, hence I went for the top level variant